scripts/jenkins/Jenkinsfile

```diff
@@ -351,8 +352,17 @@ def build_todo() {
     }
 }
 
+def build_check_release_notes() {
+    return withDockerEnv("check_release_notes", DOCKER_IMAGES.stretch) {
```

Comment:
currently i use dashes instead of _ to separate words here. so it looks a bit inconsistent.